### PR TITLE
CI Support for Windows

### PR DESCRIPTION
**Pull request by yunstanford**
Will add badge and everything else after we grant appveyor access to `huge-success/sanic`

### CODE REVIEW

File: tests/test_request_timeout.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def __init__(self, app, loop, request_delay=1):
                 return response
 
 
-Config.REQUEST_TIMEOUT = 2
+Config.REQUEST_TIMEOUT = 0.6
```
Comment (yunstanford) [inline tests/test_request_timeout.py]: 2s seems long and failed the unit test with `ClientOS` exception.

**Comment by ahopkins** [comment]
Done. forgive my windows ignorance.... what is appveyor?

**Comment by harshanarayana** [comment]
@ahopkins I've always considered it as `travis` for windwos 🤣

**Comment by yunstanford** [comment]
@ahopkins Thanks! do you mind also taking a look this pr

**Comment by ahopkins** [comment]
Will take a look later tonight.

**Comment by yunstanford** [comment]
I've squashed last couple of commits. @ahopkins

**Pull request merged**